lntest/itest/lnd_test_list_on_test.go

```diff
@@ -8,6 +8,10 @@ var allTestCases = []*testCase{
 		name: "open channel reorg test",
 		test: testOpenChannelAfterReorg,
 	},
+	{
```

Comment:
question: could you explain why you added this test to the "old tests" not to the new `list_on_test.go`. I still not really understand whats the new one or whats the difference between them both, maybe someone can clarify

Reply:
@yyforyongyu, maybe that's a question for you?

Reply:
It seems like there is a test migration going on from `lnd_test_list_on_test.go` to `list_on_test.go` which requires changes in test framework signatures and functions. @yyforyongyu to confirm.
I need to migrate the tests first before I can continue to work on your nits.

Reply:
There was an itest migration to `list_on_test.go` which uses a new function signature, so this itest needs to be changed :/

Reply:
I have integrated this test into the new test framework and will submit it shortly.